scripts/sync.py

```diff
@@ -155,6 +156,9 @@ def do(args):
             for Module in args.source:
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
+    # annonymize PII
+    dpanonymize.lock_lochness(Lochness)
```

Comment:
`dpanonymize.anon_lochness` can be a better name :)

Reply:
`lock` sounds fine to me. If you insist to go with anon_lochness, I can change.